pdsClient/pdsClient.go

```diff
@@ -245,6 +245,33 @@ func (c *E3dbPDSClient) ProxyListRecords(ctx context.Context, authToken string,
 	return result, e3dbClients.FlatMapInternalError(internalError)
 }
 
+// BatchGetRecords makes a call to batch get the records in params,
+// returning the records (if they exist) and error (if any).
+func (c *E3dbPDSClient) BatchGetRecords(ctx context.Context, params BatchGetRecordsRequest) (*BatchGetRecordsResult, error) {
+	var result *BatchGetRecordsResult
+	path := c.Host + "/" + PDSServiceBasePath + "/records/batch"
```

Comment:
would this endpoint better live at `GET /records` instead of an additional sub-path that is not win URL namespace competition with `/records/<record-id>`? Also seems like this can be the genesis of fully deprecating v1/serach and moving it to a list service, available at `GET /records`?

Is the request payload in the body or URL params?

Reply:
Good call on `GET /records`. 
Yeah, I think between this and v2 search we could deprecate v1 search and not lose any features for the customer.
Request payload is in the body.